src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -208,10 +191,36 @@ class CompanyStep extends React.Component {
                     <AddressSearch
                         label={this.props.translate('common.companyAddress')}
                         containerStyles={[styles.mt4]}
-                        value={this.getFormattedAddressValue()}
+                        value={this.state.addressStreet}
                         onChangeText={(fieldName, value) => this.clearErrorAndSetValue(fieldName, value)}
                         errorText={this.getErrorText('addressStreet')}
                     />
+                    <View style={[styles.flexRow, styles.mt4]}>
+                        <View style={[styles.flex2, styles.mr2]}>
+                            <ExpensiTextInput
+                                label={this.props.translate('common.city')}
+                                onChangeText={value => this.clearErrorAndSetValue('addressCity', value)}
+                                value={this.state.addressCity}
+                                errorText={this.getErrorText('addressCity')}
+                                translateX={-124} // TODO: Is this necessary?
```

Comment:
This prop is usually used for native platforms so the label value animates to the correct place when focusing and unfocusing the input. You can compare how it behaves with other inputs to see the difference when it isn't set, but 124 seems like a super high value here.

Reply:
Lol, it used to be 14 I think, I messed up the value because I was playing with it to see if there was any difference.

Thanks for the explanation!